### PR TITLE
chore(developer): kmc ldml build - call c8 directly for tests

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/kmc-ldml/build.sh [hunk 1]
```diff
@@ -74,7 +74,11 @@ fi
 #-------------------------------------------------------------------------------------------------------------------
 
 if builder_start_action test; then
-  npm test
+  eslint .
+  cd test
+  tsc -b
+  cd ..
+  c8 --reporter=lcov --reporter=text mocha "${builder_extra_params[@]}"
```
Comment (darcywong00) [inline developer/src/kmc-ldml/build.sh:81]: What kind of extra_params could there be?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: developer/src/kmc-ldml/build.sh (hunk 1 above)
Comment (mcdurdin) [reply developer/src/kmc-ldml/build.sh:81]: Can pass a specific test to run (i.e. `build.sh test -- my_test_name`)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.258-alpha](https://keyman.com/downloads/releases/alpha/17.0.258)

**Pull request merged**